projects/canopy/src/lib/promo-card/variables.scss

```diff
@@ -0,0 +1,2 @@
+// this has been added as a SASS variable as we cannot use CSS custom properties in media queries
+$promo-card-inline-breakpoint: 37.5rem;
```

Comment:
can this go in `components/_promo-card.scss`?

Reply:
I added it here so it's scoped to the `promo-card` module. I can move it to the `_promo-card.scss` if it's more manageable that way, though.

Reply:
When chatting about this originally, I wasn't sure about having it here or in the variables file, as it's SASS var, not a CSS custom property, and I wasn't sure if users of Canopy would assume that you can change it because it's grouped up with the ones you can override for theming (without needing to compile any SASS). On the other hand, it's still a variable and we do have a pattern for storing vars somewhere - so maybe let's put in with the other ones to keep things consistent? I remember thinking that if it comes up in the PR we should probably move it 🙂